Cargo.lock

```diff
@@ -146,7 +141,7 @@ source = "registry+https://github.com/rust-lang/crates.io-index"
 [[package]]
 name = "chalk-engine"
 version = "0.9.0"
-source = "git+https://github.com/rust-lang/chalk.git#17fb07e43b23d721f9c1adba289eac71b25f38ef"
+source = "git+https://github.com/rust-lang/chalk.git#b92c15327f1d336fec7573c7de323ab247cca386"
```

Comment:
@flodiebold what do you think should our update policy for chalk be? Just bump it liberally if it doesn't break?

Reply:
Yeah, I'd say so.